src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+
+import './TodoList.scss';
+
+type Props = {
+  preparedTodos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ preparedTodos = [] }) => (
+  <ul className="todo-list">
+    {preparedTodos.map(preparedTodo => (
```

Comment:
make it easy to read
```suggestion
  todos: Todo[];
};

export const TodoList: React.FC<Props> = ({ todos }) => (
  <ul className="todo-list">
    {todos.map(todo => (
```